doc/user_guide/features/index.rst

```diff
@@ -0,0 +1,34 @@
+.. _features:
+
+Features
+========
+
+.. toctree::
+    :maxdepth: 2
+
+    collecting_metrics
+    creating_a_release
+
+Uniform Project Layout
+----------------------
+
+The PTB expects a default project layout following "convention over configuration" wherever possible and reasonable.
+See the cookie-cutter project template, which can be found in directory `project-template`, for more details.
+For more details on this, please check out section :ref:`Getting Started` section.
```

Comment:
"Uniform Project Layout" should probably go somewhere else.
Maybe out of scope for the current PR?